qt/python/mantidqt/mantidqt/widgets/sliceviewer/peaksviewer/model.py

```diff
@@ -118,8 +118,7 @@ def slicepoint(self, selected_index, slice_info, frame):
         frame_to_slice_fn = self._frame_to_slice_fn(frame)
         peak = self.ws.getPeak(selected_index)
         slicepoint = slice_info.slicepoint
-        iz_peak = slice_info.adjust_index_for_preceding_nonq_dims(slice_info.z_index)
```

Comment:
I haven't tested this again sorry, but I'm a bit concerned about the deletion of this line. Is there a reason we no longer need to get the `iz_peak` but instead hard-code the index [2]? The code would imply that this is necessary if there are non-q dimension that are not the last dimension (i.e. that precedes the q dimension along the slice point).
Has this been tested on such workspaces (e.g. HKEL?)? If not I can do so tomorrow!

Reply:
I changed the next line to call `SliceInfo.transform` which handles the non-q dims, and it's harded coded to `2` because that is the out of plane coordinate as returned by `SliceInfo.transform`

Reply:
I see, thanks for the explanation - sorry for holding this fix up!